splitter/splitter.bal

```diff
@@ -0,0 +1,44 @@
+import ballerina/http;
+import ballerina/mime;
+import ballerina/url;
+
+type ReminderRequest record {
+    string date;
+    Event[] events;
+};
+
+type Event record {|
+    string eventName;
+    Attendee[] attendees;
+|};
+
+type Attendee record {|
+    string name;
+    string number;
+|};
+
+final string fromNo = "+15005550006";
+final string twilioSID = "VAC1829a53d52f41b4b2b1cc003c0026aa8";
+final string  apiVersion = "2010-04-01";
+
+final http:Client twilioClient = check new ("http://api.twilio.com.balmock.io");
+
+service /events on new http:Listener(8080) {
+    resource function post reminder(@http:Payload ReminderRequest request) returns error? {
+            error?[][] _ = from Event event in request.events
+            select from Attendee attendee in event.attendees
+            select sendMessage(attendee.name, event.eventName, request.date, attendee.number);
```

Comment:
Try using `do` block, since we don't need the result.

Reply:
This would give two `do` blocks like the following
```ballerina
from Event event in request.events
     do {
          from Attendee attendee in event.attendees
               do {
                    check sendReminder(attendee, event.eventName, request.date);
               };
     };
```

or the following is also possible,
```ballerina
from Event event in request.events
     do {
          _ = from Attendee attendee in event.attendees
          select sendReminder(attendee, event.eventName, request.date);
     };
```
WDYT?